### DISCUSSION TITLE
Função com retorno vazio e parâmetro por referência

### DISCUSSION THREAD

**Discussion by Dylan-Alves** [dgadelha/Portugol-Webstudio#157] [Ideias e sugestões]
Não estou conseguindo utilizar a função com referencia , segue exemplo de teste.

programa {
  funcao inicio() {
    inteiro n1 = 50

    escreva("O numero chamado e:",n1)
    novo(n1)
    escreva(" O numero agora tem um novo valor",n1)
  }

  funcao novo(inteiro &n2)
  {
    n2=n2+10
    escreva("\nNumero: ")
  }
}

**Comment by MelAndBug** [comment]
Olá fiz o teste de seu programa e esta a funcionar corretamente.
Tem que ter en conta que : 
programa {
funcao inicio() {
inteiro n1 = 50// valor de N1
escreva("O numero chamado e:",n1)-->Imprime valor 50
novo(n1)//Chamada a função
escreva(" O numero agora tem um novo valor",n1)-->Valor de n1 neste ponto e 60
}

funcao novo(inteiro &n2)//recebe o valor 50
{
n2=n2+10 //50 +10 --> novo valor de n2 = 60
escreva("\nNumero: ") // mostra em tela Numero
}
}
